spec/models/user_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
I recommend we also add specs for the instance method `all_friend_posts` that checks whether or not calling this method will result in only all the posts of the `user` 's friends